packages/cubejs-crate-driver/README.md

```diff
@@ -0,0 +1,16 @@
+<p align="center"><a href="https://cube.dev"><img src="https://i.imgur.com/zYHXm4o.png" alt="Cube.js" width="300px"></a></p>
+
+[Website](https://cube.dev) • [Docs](https://cube.dev/docs) • [Blog](https://cube.dev/blog) • [Slack](https://slack.cube.dev) • [Discourse](https://forum.cube.dev/) • [Twitter](https://twitter.com/thecubejs)
+
+[![npm version](https://badge.fury.io/js/%40cubejs-backend%2Fserver.svg)](https://badge.fury.io/js/%40cubejs-backend%2Fserver)
+[![GitHub Actions](https://github.com/cube-js/cube.js/workflows/Build/badge.svg)](https://github.com/cube-js/cube.js/actions?query=workflow%3ABuild+branch%3Amaster)
+
+# Cube.js Postgres Database Driver
```

Comment:
This obviously needs to be updated for Crate. Could you see one of the more recent community-contributed drivers' readme such as https://github.com/cube-js/cube.js/tree/master/packages/cubejs-mysql-aurora-serverless-driver (esp. including the [support section](https://github.com/cube-js/cube.js/tree/master/packages/cubejs-mysql-aurora-serverless-driver#support))?

Reply:
No problem, updating this now